default/stringtables/ja.txt

```diff
@@ -0,0 +1,15224 @@
+#
+# Notes: to avoid potential conflict with functional keys in other files, do not
+# make any stringtable keys beginning with "FUNCTIONAL_".
+# New Sitreps priorities should be added to
+# `default/customizations/common_user_customizations.txt`.
+
+# semi-randomly collected characters to force font code page loading
+# 肛門オーガズム
+# åřžßąłżęЗыдШит
+# 를와ẵừổ
+# جامعة
+# یای مجهو
+
+
+##
+## Common phrases
+##
+
+OK
+わかりました
+
+APPLY
+適用
+
+CANCEL
+キャンセル
+
+DONE
+完成です
+
+SAVE
+保存
+
+DELETE
+削除
+
+OPEN
+開く
+
+OR
+または
+
+YES
+はい
+
+NO
+いいえ
+
+# Used as a placeholder for unexpanded content in the combat log window.
+ELLIPSIS
+...
+
+EMPIRE
+帝国
+
+PLAYER
+プレーヤー
+
+HUMAN_PLAYER
+人間
+
+AI_PLAYER
+Ai
+
+ADD_AI_PLAYER
+AI を追加する
+
+OBSERVER
+オブザーバー
+
+MODERATOR
+モデレーター
+
+NO_PLAYER
+なし
+
+NO_PLAYERS
+プレイヤーなし
+
+NO_EMPIRE
+帝国なし
+
+DROP_PLAYER
+ドロップ
+
+ALL
+すべての
+
+NONE
+なし
+
+RENAME
+変更
+
+ENTER_NEW_NAME
+新しい名前を入力してください
+
+NOTHING_VALUE_SYMBOL
+--
+
+UNKNOWN_VALUE_SYMBOL
+?
+
+UNKNOWN_VALUE_SYMBOL_2
+new
+
+ERROR
+エラー
+
+RESET
+リセット
+
+NEXT
+次に
+
+BACK
+戻る
+
+LAST
+前の
+
+PAUSE
+一時 停止
+
+PAUSED
+一時 停止
+
+RESUME
+再開
+
+DIFFICULTY
+困難
+
+X
+X ポジション
+
+Y
+Y ポジション
+
+INVALID_POSITION
+不明な位置
+
+# Name for a newly created general purpose fleet.
+# %1% represents a unique number.
+NEW_FLEET_NAME
+フリート %1%
+
+# Name for a newly created general purpose fleet when no uniqe number could be
+# created.
+NEW_FLEET_NAME_NO_NUMBER
+艦隊
+
+# Name for a newly created monster-only fleet.
+# %1% represents a unique number.
+NEW_MONSTER_FLEET_NAME
+群れ %1%
+
+# Name for a newly created colony-only fleet.
+# %1% represents a unique number.
+NEW_COLONY_FLEET_NAME
+移民艦隊 %1%
+
+# Name for a newly created reconnaissance-only fleet.
+# %1% represents a unique number.
+NEW_RECON_FLEET_NAME
+偵察艦隊 %1%
+
+# Name for a newly created troop carrier only fleet.
+# %1% represents a unique number.
+NEW_TROOP_FLEET_NAME
+部隊艦隊 %1%
+
+# Name for a newly created bomber only fleet.
+# %1% represents a unique number.
+NEW_BOMBARD_FLEET_NAME
+爆撃機艦隊 %1%
+
+# Name for a newly created combat group group fleet. This includes battleships,
+# bombers and troop carriers.
+# %1% represents a unique number.
+NEW_BATTLE_FLEET_NAME
+戦闘艦隊 %1%
+
+# Name for a newly created planet.
+# Suffix bears some explanation:
+# - Planets are grouped for asteroids and non-asteroids.
+# - Suffix is a roman numeral, with additional rules for asteroids.
+# - The roman numeral is a rank for proximity to the center of the system,
+# in relation to other planets in the same group.
+# - For asteroids, the suffix starts with a localized NEW_ASTEROIDS_SUFFIX.
+# If any other asteroids are in the system, the roman numeral is appended.
+# %1% name of the system this planet is created in.
+# %2% suffix for this planet
+NEW_PLANET_NAME
+%1% %2%
+
+# The label pre-pended to a new asteroids naming suffix
+
+
+
+EMPTY_SPACE
+深宇宙
+
+UNEXPLORED_REGION
+未踏地域
+
+UNEXPLORED_SYSTEM
+未踏システム
+
+NOT_APPLICABLE
+N/a
```

Comment:
Identical to en translation, leave the key out.

Reply:
I have some python script that can do it.  I don't think this should be done manually. 

@ganbaranaiyo are you comfortable on a running script in the console?